### PR TITLE
Linked custom entities

### PR DESCRIPTION
**Pull request by rybus**

### CODE REVIEW

File: src/Acme/Bundle/CustomBundle/Manager/ChoicesManager.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+/**
+ * Choices manager
+ * This class is used to retrieve the list all entries for en entity.
+ * This list is used in the filter form for custom entities datagrid (i.e: Brand).
+ *
+ * Please note that this can be used for all custom entities.
+ *
+ * @author Remy Betus <remy.betus@akeneo.com>
+ */
+class ChoicesManager
```
Comment (fitn) [inline src/Acme/Bundle/CustomBundle/Manager/ChoicesManager.php]: Please rename it

File: src/Acme/Bundle/CustomBundle/Form/Type/BrandType.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        parent::buildForm($builder, $options);
+
+        $builder
+            ->add('fabric', 'entity', [
+                'class'        => 'AcmeCustomBundle:Fabric',
+                'choice_label' => 'name',
+                'required'     => false,
+                'placeholder'  => 'Choose a fabric',
+                'empty_data'   => null
+            ])
```
Comment (fitn) [inline src/Acme/Bundle/CustomBundle/Form/Type/BrandType.php:35]: Will break phpcs

File: src/Acme/Bundle/CustomBundle/Form/Type/BrandMassEditType.php [hunk 3]
```diff
@@ -16,7 +16,15 @@ class BrandMassEditType extends AbstractType
...
     {
-        $builder->add('code', null, ['select2' => true]);
+        $builder
+            ->add('fabric', 'entity', [
+                'class'        => 'AcmeCustomBundle:Fabric',
+                'choice_label' => 'name',
+                'required'     => false,
+                'placeholder'  => 'Choose a fabric',
+                'empty_data'   => null
+            ])
```
Comment (fitn) [inline src/Acme/Bundle/CustomBundle/Form/Type/BrandMassEditType.php:26]: will break phpcs

**Comment by fitn** [comment]
Some comments then GTM

File: src/Acme/Bundle/CustomBundle/Form/Type/BrandType.php (hunk 2 above)
Comment (rybus) [inline src/Acme/Bundle/CustomBundle/Form/Type/BrandType.php:35]: no :P

File: src/Acme/Bundle/CustomBundle/Form/Type/BrandMassEditType.php (hunk 3 above)
Comment (rybus) [inline src/Acme/Bundle/CustomBundle/Form/Type/BrandMassEditType.php:26]: same here

**Comment by fitn** [comment]
Just wait the 1.4.17 release and update the composer.lock

**Pull request merged**